test-app/package.json

```
@@ -96,7 +96,7 @@
     "stylelint-config-standard": "^34.0.0",
     "stylelint-prettier": "^4.0.2",
     "tracked-built-ins": "workspace:*",
-    "typescript": "~4.9.5",
+    "typescript": "5.6.3",
```

let's keep tilde to allow patch version to drift
```suggestion
    "typescript": "~5.6.3",
```